pkgs/by-name/th/thc-secure-delete/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenvNoCC # x86_64-darwin and aarch64-darwin cant find C compiler
+, fetchFromGitHub
+, pkg-config
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "thc-secure-delete";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gordrs";
+    repo = "thc-secure-delete";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-hvWuxFkzhOSCplPtyjRtn36bIk6KdPBcpr3lAmiAyfE=";
+  };
+
+  nativeBuildInputs = [ pkg-config gcc ];
+
+  buildPhase = ''
+    substituteInPlace Makefile --replace "gcc" "${gcc}/bin/gcc"
+    make
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 srm $out/bin
+    install -Dm755 sfill $out/bin
+    install -Dm755 sswap $out/bin
+    install -Dm755 sdmem $out/bin
+    install -Dm755 sfill.1 $out/share/man/man1/sfill.1.gz
+    install -Dm755 srm.1 $out/share/man/man1/srm.1.gz
+    install -Dm755 smem.1 $out/share/man/man1/smem.1.gz
+    install -Dm755 sswap.1 $out/share/man/man1/sswap.1.gz
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
`meta.platforms` is missing, probably should be set to `lib.platforms.unix` or `lib.platforms.all`.